app/views/shopping_lists/player_shopping_list.html.erb

```diff
@@ -15,3 +15,5 @@
     </div>
   <% end %>
 </div>
+
+<%= link_to "Back", root_path %>
```

Comment:
Sorry, this is the one I meant. It will bring the user to whatever page they came from
```suggestion
<%= button_to "Back", :back, class: "button-b", method: :get %>
```